Topic 1/src/com/globant/topic1/exercise1/model/Email.java

```diff
@@ -0,0 +1,56 @@
+package com.globant.topic1.exercise1.model;
+
+public class Email {
+	// Final Fields
+	private static final String TO_EMAIL = "manager@email.com";
```

Comment:
This should be called MANAGER_EMAIL to know which one are we goint to send the email.